### PR TITLE
Implement Login

### PR DESCRIPTION
**Pull request by Merzarus**

### CODE REVIEW

File: src/components/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+              </Button>
+            </Row>
+          </Col>
+        </Row>
+      </Container>
+
+
+    </Form>
+  )
+}
```
Comment (themaxhero) [inline src/components/LoginForm.tsx]: Please, put a empty new line in the end of this file.
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,can%20handle%20the%20EOF%20marker.

File: src/App.tsx [hunk 2]
```diff
@@ -1,27 +1,12 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <Login></Login>
```
Comment (Amadeusvh) [inline src/App.tsx]: Pode ser trocado por ```<Login />```

File: src/index.css [hunk 3]
```diff
@@ -11,3 +11,9 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+body, html, #root {
+  width: 100%;
+  height: 100%;
+  background-color: #323232;
+}
```
Comment (themaxhero) [inline src/index.css]: Please, put a empty new line in the end of this file.
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,can%20handle%20the%20EOF%20marker.

File: src/pages/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        <Col md="4" className="ml-6 d-flex justify-content-sm-around" lg="4"><h1>Moneyman</h1></Col>
+      </Row>
+      <Row className="mt-5">
+        <Col className="align-self-end">
+          <LoginForm></LoginForm>
+        </Col>
+      </Row>
+    </Container>
+  )
+}
```
Comment (themaxhero) [inline src/pages/Login.tsx]: Please, put a empty new line in the end of this file.
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,can%20handle%20the%20EOF%20marker.

File: src/pages/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+type LoginProps = {
+  optional?: string;
+}
+
+export default (props: LoginProps) => {
+  return (
+    <Container className="d-flex align-content-between justify-content-sm-between flex-column h-100 p-5">
+      <Row className="d-flex justify-content-center mt-5 mb-5">
+        <Col md="4" className="ml-6 d-flex justify-content-sm-around" lg="4"><h1>Moneyman</h1></Col>
```
Comment (themaxhero) [inline src/pages/Login.tsx]: Please break line before and after h1.

**Review by themaxhero — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 6]
```diff
@@ -1,27 +1,12 @@
...
+  )
 }
 
 export default App;
```
Comment (jvrmalv) [inline src/App.tsx:12]: Please, put a empty new line in the end of this file.
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,can%20handle%20the%20EOF%20marker.

**Review by themaxhero — APPROVED** [review]

**Review by jvrmalv — APPROVED** [review]

**Review by Amadeusvh — APPROVED** [review]

**Pull request merged**